### DISCUSSION TITLE
HEIC image support

### DISCUSSION THREAD

**Discussion by jayasio** [notaapp/talk#400] [Feature Requests]
I'm really enjoying using Nota!

I'd like to ask, could Nota support HEIC image previews? It would be convenient for me to include paper sketches in my notes by just taking a picture, Airdrop to my Mac, and adding it to Nota, without needing to convert to JPG/PNG.

Also, it would be great if Nota could incorporate the 'insert from iPhone/iPad' feature to simplify the process of adding photos or iPad sketches to the notes. Like Apple Notes, could Nota have this feature, or is it exclusive to first-party apps?

**Comment by astoilkov** [comment]
Hi.

> I'd like to ask, could Nota support HEIC image previews? It would be convenient for me to include paper sketches in my notes by just taking a picture, Airdrop to my Mac, and adding it to Nota, without needing to convert to JPG/PNG.

Yeah, I think this is possible. Have you seen any app do it? I guess Apple Notes might be doing it, anything else? Thanks.

> Also, it would be great if Nota could incorporate the 'insert from iPhone/iPad' feature to simplify the process of adding photos or iPad sketches to the notes. Like Apple Notes, could Nota have this feature, or is it exclusive to first-party apps?

I'm not sure about that. I will have to take a look at it.

**Comment by jayasio** [comment]
I tried with Obsidian, but it doesn't preview HEIC images either. Apple Notes has the capability; haven't been able to try out other apps.
